### ISSUE TITLE
How can I use Microket for promoter capture Hi-C?

### ISSUE THREAD

**Issue by xiasijian** [hellosunking/Microcket#2]
Thanks for your excellent software, Could you provide the benchmark script for promoter capture Hi-C  dataset as your paper has described?

**Comment by hellosunking** [comment]
Yes you can, just run Microcket on it as other datasets. In fact, capture Hi-C datasets were included in the benchmark tests in our paper, you may check it if you are interested.

**Comment by xiasijian** [comment]
Thanks a lot, I will check it.
